paymentsheet/src/main/java/com/stripe/android/common/model/CommonConfiguration.kt

```diff
@@ -83,6 +95,11 @@ internal data class CommonConfiguration(
     }
 }
 
+@RestrictTo(RestrictTo.Scope.LIBRARY)
```

Comment:
This shouldn't need a restrict to -- afaik we only need this for public functions to hide them, but for internal functions, we don't need to add this